go/cmd/aggregate-crls/aggregate-crls_test.go

```diff
@@ -32,5 +43,166 @@ func Test_makeFilenameFromUrl(t *testing.T) {
 		"http://crl.repository.com/",
 		"http://crl.repository.com/crl"}
 	checkCollision(t, crls2, names)
+}
+
+func makeCA(t *testing.T) (*x509.Certificate, interface{}) {
+	caTemplate := &x509.Certificate{
+		SerialNumber: big.NewInt(1),
```

Comment:
With a fixed serial, this will create a lot of certificates with the same issuer/serial but different contents. I guess it's not a big deal if these aren't persistent and don't interact with NSS ever.

Reply:
I mean, might as well use the current epoch timestamp 👍 =D